.github/workflows/tests.yml

```diff
@@ -15,7 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        part: ["00", "01", "02", "03", "04", "05"]
+        repeat: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
```

Comment:
Good! Now that tests are unflaked, can we go back to not running them on repeat? This will help keep CI budget under control when testing our day-to-day changes. If we ever run into flakiness again, we can always repeat the experiment in a draft PR.

Reply:
How much is CI budget vs developer time spent guessing which PR to merge?

Reply:
IDK, grinding all tests redundantly N times just in case we might reveal another flaky test a few pushes earlier sounds excessive to me. There's also N times the reporting, we could do with less visual noise from CI as it is.